modules/ROOT/pages/enable-verbose-logging.adoc

```diff
@@ -0,0 +1,68 @@
+= Enable Verbose Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can enable verbose logging for different Modules and Connectors. Because of the high impact caused in application performance, verbose logging must be enabled only for testing and troubleshooting purposes.
+
+== Configure Verbose Logging
+
+To configure verbose logging, add an entry to your `log4j2.xml` indicating the package names of the Connectors or Modules you want to log, and set the log level to `DEBUG`.
+
+The following example shows how to find the package names for the JMS Connector and configure verbose logging:
+
+. In the *Package Explorer* view, open the JMS Connector package library
+. Inside the `.jar` file, locate all Mule related packages
++
+image:mruntime-verbose-log-package.png[verbose-logging-package]
+. Add an entry to your `log4j2.xml` file, indicating the package name and log level = `DEBUG`:
++
+[source,xml,linenums]
+----
+<Loggers>
+    ...
+    <AsyncLogger name="org.mule.extensions.jms" level="DEBUG"/>
+    ...
+</Loggers>
+----
+
+== Common Connectors Package Names
+
+The following table shows some common Connectors and their package names so you don't have to search for them:
```

Comment:
The following table shows some **common Anypoint Connectors** and their package names so you don't have to search for them:

Reply:
All of the connector names that follow should be presented as "Anypoint Connector for <this>," per our style guide and PMM advice.
See the style guide entry for "connector naming."
I'll let you change these.  :-)